src/core_plugins/timelion/server/handlers/__tests__/parse_sheet.js

```diff
@@ -0,0 +1,50 @@
+const chainRunnerFn = require('../chain_runner');
+const tlConfig = require('../../series_functions/__tests__/fixtures/tlConfig.js')();
+
+const expect = require('chai').expect;
+
+describe('timelion parse_sheet function', function () {
+  const chainRunner =  chainRunnerFn(tlConfig);
+
+  async function parse(data) {
+    const request = {
+      sheet: data,
+      time: {
+        from: 'now-15m',
+        to: 'now',
+        mode: 'quick',
+        interval: 'auto',
+        timezone: 'Europe/Berlin'
+      }
+    };
+    return await Promise.all(chainRunner.processRequest(request));
```

Comment:
This doesn't belong in this file, as it is testing the interpreter. You want to test that the grammar is generating a parser that is producing the expected AST

This will make the tests run faster, as well as make the tests more useful to the reader by conveying that we are only testing the parser.

This will also keep you from having to mock the `time` property.